hedera-node/src/test/java/com/hedera/services/context/properties/GlobalDynamicPropertiesTest.java

```diff
@@ -97,6 +97,7 @@ void constructsFlagsAsExpected() {
         assertTrue(subject.requireMinStakeToReward());
         assertTrue(subject.isTraceabilityMigrationEnabled());
         assertFalse(subject.shouldCompressRecordFilesOnCreation());
+        assertTrue(subject.shouldCompressAccountBalanceFilesOnCreation());
```

Comment:
The pending [sonar smell](https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=3952&id=com.hedera.hashgraph%3Ahedera-services&open=AYNWDi20RVNptEeyg_YC) is because of too many assertions in this method... Not sure if we can do anything? A couple of methods in this test class have the same problem.

Reply:
Hi @dimitar-dinev, yes you can disable this warning at the source via,
```
    @Test
    @SuppressWarnings("java:S5961")
    void reloadsFlagsAsExpected() {
        ....
```

Reply:
👍🏽  Added `@SuppressWarnings("java:S5961")`  on the methods that had the problem